scripts/release-build.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+
+const fs = require('fs').promises;
+const path = require('path');
+
+const directory = './build/';
+
+const verbatimFiles = ['LICENSE', 'README.md', 'index.d.ts', 'types.d.ts'];
+
+// Returns a string representing data ready for writing to JSON file
+function createDataBundle() {
+  const bcd = require('../index.js');
+  const string = JSON.stringify(bcd);
+  return string;
+}
+
+// Returns a promise for writing the data to JSON file
+function writeData() {
+  const dest = path.resolve('./', directory, './data.json');
+  const data = createDataBundle();
+  const promise = fs.writeFile(dest, data);
+  return promise;
+}
+
+// Returns an array of promises for copying of all files that don't need transformation
+function copyFiles() {
+  let promises = [];
+  for (const file of verbatimFiles) {
+    const src = path.join('./', file);
+    const dest = path.join(directory, file);
+    const promise = fs.copyFile(src, dest);
+    promises = promises.concat(promise);
+  }
+  return promises;
+}
+
+function createManifest() {
+  const full = require('../package.json');
+  const minimal = {};
+
+  const allowList = [
+    'name',
+    'version',
+    'description',
+    'repository',
+    'keywords',
+    'author',
+    'license',
+    'bugs',
+    'homepage',
+    'types',
+  ];
+
+  for (const key of allowList) {
+    if (key in full) {
+      minimal[key] = full[key];
+    } else {
+      throw `Could not create a complete manifest! ${key} is missing!`;
+    }
+  }
+  return JSON.stringify(minimal);
+}
+
+function writeManifest() {
+  const dest = path.resolve('./', directory, './package.json');
+  const manifest = createManifest();
+  const promise = fs.writeFile(dest, manifest);
+  return promise;
+}
+
+async function main() {
+  // Remove existing files, if there are any
+  const deletedDir = await fs.rm(directory, {
+    force: true,
+    recursive: true,
+  });
+  if (deletedDir !== undefined) throw deletedDir;
+
+  // Crate a new directory
+  const createdDir = await fs.mkdir(directory);
+  if (createdDir !== undefined) throw createdDir;
+
+  const promises = [writeManifest(), writeData(), ...copyFiles()];
+  let errors = await Promise.all(promises);
+  errors = errors.filter(e => e !== undefined);
+  if (errors.length !== 0) {
+    // Re-throw all errors
+    throw errors;
+  }
```

Comment:
```suggestion
  await writeManifest();
  await writeData();
  await copyFiles();
  
  console.log('Data bundle is ready');
```

Suggesting making this as simple as possible, without parallelism.

Reply:
OK. We can always make it more parallel later if there is need for that. If I recall correctly, there was a large speedup from using async I/O in general, but impact of awaiting for `writeManifest`, `writeData` and `copyFiles` in parallel was measureable only on external HDD connected via USB2.0.

Reply:
@bershanskiy I thought that your measurements were for loading the resulting package, not for preparing it? I think it would be fine if running this script took a long time, since it's only done by maintainers. It doesn't seem to actually be slow, however.

If one wanted to speed this up by parallelizing I/O, I think the loading of the JSON files in `index.js` is where the wins would be.